plone/app/content/browser/vocabulary.py

```diff
@@ -183,6 +184,8 @@ def __call__(self):
         ]
         if attributes:
             base_path = getNavigationRoot(context)
+            sm = getSecurityManager()
+            can_edit = sm.checkPermission('Modify portal content', context)
             for vocab_item in results:
```

Comment:
@thet Hi, I'm a bit confused. In folder_contents, the `getVocabulary` view is called on the portal. Therefore the context is the portal.
Why does a user need to have "Modify portal content" globally on the portal to be able to view the Creator column in any subfolder?
Why is Creator considered unsafe in the first place?